src/main/java/net/imglib2/view/Views.java

```diff
@@ -1234,4 +1235,19 @@ public static boolean isZeroMin( final Interval interval )
 		return Views.interval( new TransformView< T >( source, transform.inverse() ), source );
 	}
 
+	/**
+	 * Compose two {@link RandomAccessible} sources into a
+	 * {@link RandomAccessible} of {@link Pair}.
+	 *
+	 * @param sourceA
+	 * @param sourceB
+	 * @return
+	 */
+	public static < A, B > RandomAccessiblePair< A, B > pair(
```

Comment:
Thanks.  Fixed with making the `RandomAccess` implement `Pair` as suggested below.